lib/jsdom/living/nodes/HTMLTextAreaElement-impl.js

```diff
@@ -69,12 +71,14 @@ class HTMLTextAreaElementImpl extends HTMLElementImpl {
   }
 
   set value(val) {
+    const oldValue = this._rawValue;
     this._rawValue = val;
     this._dirtyValue = true;
 
-    this._selectionStart = 0;
-    this._selectionEnd = 0;
-    this._selectionDirection = "none";
+    if (oldValue !== this._rawValue) {
```

Comment:
This doesn't seem to match the spec; see https://html.spec.whatwg.org/multipage/form-elements.html#dom-textarea-value. You need to use API value instead of raw value.

Reply:
Thank you this is very helpful.  Looks like this may fix an additional test as well.  Working on it.

Reply:
This has been fixed, and now got `selection-after-content-change` to pass.